lenthall/README.md

```diff
@@ -0,0 +1,27 @@
+[![Build Status](https://travis-ci.org/broadinstitute/lenthall.svg?branch=develop)](https://travis-ci.org/broadinstitute/lenthall?branch=develop)
+[![codecov](https://codecov.io/gh/broadinstitute/lenthall/branch/develop/graph/badge.svg)](https://codecov.io/gh/broadinstitute/lenthall)
+
+Lenthall
+========
+
+[Cromwell](https://github.com/broadinstitute/cromwell) Common Code
+
+# Mailing List
+
+The [Cromwell Mailing List](https://groups.google.com/a/broadinstitute.org/forum/?hl=en#!forum/cromwell) is cromwell@broadinstitute.org.
+
+If you have any questions, suggestions or support issues please send them to this list. To subscribe you can either join via the link above or send an email to cromwell+subscribe@broadinstitute.org.
```

Comment:
TOL do we want to encourage people to sign up for the mailing list? I did a quick look and we don't publicize the Cromwell mailing list anywhere else in Github. I think it's fine as-is, it might change after our Doc-A-Thon day.

Also this PR is so big that my message is slow to type because Github is breaking. Congrats @mcovarr, you broke the Crom-net.
![screen shot 2017-10-04 at 11 00 03 am](https://user-images.githubusercontent.com/2395211/31183044-7a425e64-a8f3-11e7-9d67-e85d53ec44ff.png)

Reply:
I think I just killed the mailing list the other day. If I didn't, I meant to.